src/kernels/raw/finder/jupyterPaths.node.ts

```diff
@@ -16,10 +17,20 @@ import {
 import { tryGetRealPath } from '../../../platform/common/utils.node';
 import { IEnvironmentVariablesProvider } from '../../../platform/common/variables/types';
 import { traceDecorators } from '../../../platform/logging';
+// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
+const untildify = require('untildify');
+
+const homeDir = untildify('~');
```

Comment:
I think we already have this in PathUtils.home

Reply:
yes, i felt it was unnecessary to add yet another object via DI when it's all the same.

Reply:
Nope, convinced it not the right thing to do.
Changed as requested.